TwoNote/app/src/main/res/layout/activity_main.xml

```diff
@@ -4,12 +4,13 @@
   ~  Licensed under the MIT License.
   ~
   -->
-<com.microsoft.device.dualscreen.layouts.SurfaceDuoLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<com.microsoft.device.dualscreen.layouts.SurfaceDuoLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
is this supposed to be all one line?

Reply:
i can move it!

Reply:
just kidding the autoformatter makes it like that (and I think it's similar in other layout files)